specification/resources/resource-manager/Microsoft.Resources/preview/2023-07-01-preview/examples/ListChanges.json

```diff
@@ -0,0 +1,75 @@
+{
+  "parameters": {
+    "subscriptionId": "subscriptionId1",
+    "resourceGroupName": "resourceGroup1",
+    "resourceProviderNamespace": "resourceProvider1",
+    "resourceType": "resourceType1",
+    "resourceName": "resourceName1",
+    "api-version": "2023-03-01-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/subscriptions/subscriptionId1/resourceGroups/resourceGroup1/providers/resourceProvider1/resourceType1/resourceName1/Microsoft.Resources/changes/a9f34285-13a2-e79c-f468-cfb71c7bd227",
```

Comment:
>/subscriptions/subscriptionId1/resourceGroups/resourceGroup1/providers/resourceProvider1/resourceType1/resourceName1/Microsoft.Resources/changes/a9f34285-13a2-e79c-f468-cfb71c7bd227" [](http://example.com/codeflow?start=19&length=182)

Suggest adding real resource types examples after obfuscating the names.

Reply:
I like